docs/en_us/platform_api/source/bookmarks/endpoints.rst

```diff
@@ -0,0 +1,25 @@
+.. _edX Bookmarks API Endpoints:
+ 
+################################################
+Bookmarks API Endpoints
+################################################
+ 
+You use the Bookmarks API to add a bookmark to an XBlock, get a bookmark or
+list of bookmarks for a course, and delete a bookmark.
+
+The following tasks and endpoints are currently supported.
+ 
+.. list-table::
+   :widths: 10 70
+   :header-rows: 1
+ 
+   * - To:
+     - Use this endpoint:
+   * - :ref:`Get a list of bookmarks in reverse chronological order <Get List or Create Bookmarks>`
+     - GET /api/bookmarks/v0/bookmarks/{course_id}
```

Comment:
The correct url is `GET /api/bookmarks/v1/bookmarks/?course_id={course_id1}&fields=display_name,path`
Note: `{course_id}` is query-string hence optional, just like in above url.